scripts/superpmi.py

```diff
@@ -151,6 +153,7 @@
 asm_diff_parser.add_argument("-test_env", dest="test_env", default=None)
 asm_diff_parser.add_argument("-output_mch_path", dest="output_mch_path", default=None)
 asm_diff_parser.add_argument("-run_from_coreclr_dir", dest="run_from_coreclr_dir", default=False)
+asm_diff_parser.add_argument("-previous_temp_location", dest="previous_temp_location", default=None)
```

Comment:
There are a lot of undocumented arguments here; it would be good to give them all useful documentation, with examples as necessary.

Reply:
Added a help comment.